samples/router/src/MobileLayout/ChannelSettings.tsx

```diff
@@ -1,16 +1,18 @@
 import { useNavigate, useParams } from 'react-router-dom'
-
 import SbChannelSettings from '@sendbird/uikit-react/ChannelSettings'
 
+import { useNavigateOnBan } from '../hooks/useNavigateOnBan'
+
 export function ChannelSettings() {
   const { channelUrl } = useParams()
   const navigate = useNavigate()
+  useNavigateOnBan()
   return (
-    <div>
+    <div style={{ height: '100vh' }}>
```

Comment:
Can we use `100%` instead of using `100vh`? Because it breaks some CSS.

Reply:
100% is probably cleaner, but, this is sample, doesnt matter too much